utils/serviceClient.ts

```diff
@@ -0,0 +1,76 @@
+import * as te from "fp-ts/lib/TaskEither";
+
+import {
+  IResponseErrorInternal,
+  ResponseErrorInternal
+} from "@pagopa/ts-commons/lib/responses";
+
+import { LimitedProfile } from "@pagopa/io-functions-commons/dist/generated/definitions/LimitedProfile";
+
+import { FiscalCode } from "@pagopa/ts-commons/lib/strings";
+import fetch, { Response } from "node-fetch";
+import { toError } from "fp-ts/lib/Either";
+
+export interface IServiceClient {
+  readonly getLimitedProfileByPost: (
+    reqHeaders: NodeJS.Dict<string | ReadonlyArray<string>>,
+    fiscalCode: FiscalCode
+  ) => te.TaskEither<IResponseErrorInternal, LimitedProfile>;
+  readonly submitMessageForUser: (
+    reqHeaders: NodeJS.Dict<string | ReadonlyArray<string>>,
+    reqPayload: Response
+  ) => te.TaskEither<IResponseErrorInternal, Response>;
+}
+
+export const createClient = (
+  apiUrl: string,
+  apiKey: string
+): IServiceClient => ({
+  getLimitedProfileByPost: (
+    reqHeaders,
+    fiscalCode
+  ): ReturnType<IServiceClient["getLimitedProfileByPost"]> =>
+    te
+      .tryCatch(
+        () =>
+          fetch(`${apiUrl}/v1/profiles`, {
```

Comment:
We should use a KeepAlive fetch. `getHttpsFetch` method from `@pagopa/ts-commons` returns a custom fetch with the agent_keep_alive configurable throw env variable (`FETCH_KEEPALIVE_ENABLED`).
Follow as example the configuration used in [here](https://github.com/pagopa/io-functions-app/blob/5a35d9fa4cd76b640eccce8e5dd1db04cf90698c/SendWelcomeMessagesActivity/index.ts#L19)
cc. @balanza

Reply:
Thanks, I will

Reply:
done in https://github.com/pagopa/io-functions-eucovidcerts/pull/9/commits/8b1c5d9a813d63a3823387ce4c36b87e195d866a